man/man8/zpool.8

```diff
@@ -41,8 +41,9 @@ zpool \- configures ZFS storage pools
 
 .LP
 .nf
-\fBzpool create\fR [\fB-fnd\fR] [\fB-o\fR \fIproperty=value\fR] ... [\fB-O\fR \fIfile-system-property=value\fR]
-     ... [\fB-m\fR \fImountpoint\fR] [\fB-R\fR \fIroot\fR] [\fB-t\fR \fItname\fR] \fIpool\fR \fIvdev\fR ...
+\fBzpool create\fR [\fB-fnd\fR] [\fB-o\fR \fIproperty=value\fR] ... [\fB-o\fR feature@\fIfeature=value\fR]
+     ... [\fB-O\fR \fIfile-system-property=value\fR] ... [\fB-m\fR \fImountpoint\fR] [\fB-R\fR \fIroot\fR] [\fB-t\fR \fItname\fR]
+     ... \fIpool\fR \fIvdev\fR ...
```

Comment:
Can you fix the line wrapping here.  [-t name] needs to be moved to the last line with ... pool vdev ...

```
       zpool create [-fnd] [-o property=value] ... [-o feature@feature=value]
            ... [-O file-system-property=value] ... [-m mountpoint] [-R root]
[-t tname]
            ... pool vdev ...
```